src/OpenTelemetry.Collector.Dependencies/Implementation/HttpHandlerDiagnosticListener.cs

```diff
@@ -69,12 +69,13 @@ public override void OnStartActivity(Activity activity, object payload)
 
             if (span.IsRecording)
             {
+                span.PutComponentAttribute("http");
+                span.PutHttpTargetAttribute(request.RequestUri.PathAndQuery);
```

Comment:
spec requires only one set

```
One of the following sets of attributes is required (in order of usual preference unless for a particular web client/framework it is known that some other set is preferable for some reason; all strings must be non-empty):

http.url
http.scheme, http.host, http.target
http.scheme, peer.hostname, peer.port, http.target
http.scheme, peer.ip, peer.port, http.target
```

why should we populate target, flavor and scheme? They all could be inferred from url and introduce redunduncy.

Reply:
Ok, I will remove `http.target` and `http.scheme`